tests/Unit/UseCases/ApplyForMembership/MembershipApplicationValidatorTest.php

```diff
@@ -50,12 +48,57 @@ public function testGivenInvalidAmount_validationFails( string $amount, string $
 
 	public function invalidAmountProvider() {
 		return [
-			'too low' => [ '1.00', Result::VIOLATION_TOO_LOW ],
-			'just too low' => [ '24.99', Result::VIOLATION_TOO_LOW ],
-			'max too low' => [ '0', Result::VIOLATION_TOO_LOW ],
-			'invalid: negative' => [ '-1.00', Result::VIOLATION_NOT_MONEY ],
-			'invalid' => [ 'y u no btc', Result::VIOLATION_NOT_MONEY ],
+			'invalid: negative' => [ '-1.00', 3, Result::VIOLATION_NOT_MONEY ],
```

Comment:
It's not money in the sense that you cannot give someone -1 EUR, which is a thing our Euro class checks.